plugins/mcreator-core/procedures/play_sound.json

```diff
@@ -24,6 +24,52 @@
       "type": "input_value",
       "name": "pitch",
       "check": "Number"
+    },
+    {
+      "type": "field_dropdown",
+      "name": "category",
+      "options": [
```

Comment:
Maybe a mapping file would be better for the future. It would allow us to re-use these values in other procedure blocks if needed or in parameters for mod elements.

Reply:
I agree, also soundelements use this too so that dropdown could use data lists then too